cloudbuild.yaml

```diff
@@ -0,0 +1,111 @@
+steps:
+
+# Create docker image from regular Dockerfile
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '--tag=gcr.io/google.com/hadoop-cloud-dev/init-actions-image', '.']
+# Run tests from the repo, based on the branch name
+- name: 'gcr.io/google.com/hadoop-cloud-dev/init-actions-image'
+  entrypoint: 'bash'
+  args:
+  - '-c'
+  - |
+    [[ "$BRANCH_NAME" == .*bigtable.* ]] && python bigtable/test_bigtable.py || exit 0
```

Comment:
Is it possible to make this generic instead of copy/pasting for each test/init-action?

Reply:
Removed this file in place of multiple config files. Instead of deciding whether to run a given test based on the branch name as shown here, now we handle that logic at the trigger level in GCP.